setup.py

```diff
@@ -23,31 +23,34 @@
 
 
 setup(
-    name="teachers_digital_platform",
+    name="crtool",
```

Comment:
I don't think renaming the package this way will work when installed in cfgov-refresh. For one, the cfgov/urls.py file [depends on](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/urls.py#L480) the current TDP package name.

Reply:
The `name` in `setup()` isn't necessarily the [Python package name](https://docs.python.org/3/tutorial/modules.html#packages) so much as the [name of the distribution (wheel file in our case)](https://setuptools.readthedocs.io/en/latest/setuptools.html#new-and-changed-setup-keywords). 

I think this shouldn't have any effect, but, my preference would be to do two things:

- Keep the `name` in `setup.py` the same as the repo name (with underscores instead of hyphens, of course). We could rename this repo `crtool`.
- Add `crtool.urls` to `cfgov/urls.py` in cfgov-refresh.